internal/spdxlicense/license_test.go

```diff
@@ -0,0 +1,44 @@
+package spdxlicense
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIDParse(t *testing.T) {
+	var tests = []struct {
+		shortName string
+		spdx      string
+	}{
+		{
+			"gpl-1-only",
+			"GPL-1.0-only",
+		},
+		{
+			"gpl-2+",
+			"GPL-2.0+",
+		},
+		{
+			"gpl-3.0.0-or-later",
+			"GPL-3.0-or-later",
+		},
+		{
+			"gpl-3-with-autoconf-exception",
+			"GPL-3.0-with-autoconf-exception",
+		},
+		{
+			"cc-by-nc-3-de",
+			"CC-BY-NC-3.0-DE",
+		},
+	}
+
+	for _, test := range tests {
+		got, exists := ID(test.shortName)
+		if exists {
+			assert.Equal(t, got, test.spdx)
+			return
```

Comment:
this return short circuits the tests (only one passing case is tested). With `t.Run()` the return would work out alright, but there is some room for testify to make this look even simpler:
```golang
		t.Run(test.shortName, func(t *testing.T) {
			got, exists := ID(test.shortName)
			assert.True(t, exists)
			assert.Equal(t, test.spdx, got)
		})
```

Reply:
Updated - realized this when testing suggestions from @kzantow

Reply:
simplified with newest commit